packages/experimental/Switch/src/useSwitch.ts

```diff
@@ -26,7 +32,22 @@ export const useSwitch = (props: SwitchProps): SwitchInfo => {
     ...rest
   } = props;
 
-  const [checkedState, toggleCallback] = useAsToggleWithEvent(defaultChecked, checked, onChange);
+  // Use Layout Animation for Knob animating on state change
+  const animateSwitchKnob = () => {
```

Comment:
This should probably either be wrapped in a hook or moved to just outside of the useSwitch hook so that the function is not recreated on every render.

Reply:
Moved outside useSwitch.

Reply:
Changes done and approved.